Source/WebCore/inspector/InspectorStyleSheet.cpp

```diff
@@ -296,10 +292,6 @@ class StyleSheetHandler : public CSSParserObserver {
 
 void StyleSheetHandler::startRuleHeader(StyleRuleType type, unsigned offset)
 {
-    // Pop off data for a previous invalid rule.
```

Comment:
Is this related to this patch?

Reply:
Yes. We can no longer pop off the rule data here because that would pop off the parent data for a nested rule, but we haven’t reached the end of the parent rule’s body yet.

Reply:
But also after looking further at what is changing in this patch, removing this isn't necessary.